app/assets/v2/js/cart.js

```diff
@@ -928,133 +928,207 @@ Vue.component('grants-cart', {
-
-        // Replace undefined comments with empty strings
-        const comment = donation.grant.grant_comments === undefined ? '' : donation.grant.grant_comments;
-
-        // For automatic contributions to Gitcoin, set 'gitcoin-grant-input-amount' to 100.
-        // Why 100? Because likely no one will ever use 100% or a normal grant, so using
-        // 100 makes it easier to search the DB to find which Gitcoin donations were automatic
-        const isAutomatic = donation.grant.isAutomatic;
-        const gitcoinGrantInputAmt = isAutomatic ? 100 : Number(this.gitcoinFactorRaw);
-
-        // Add the donation parameters
-        saveSubscriptionPayload.admin_address.push(donation.grant.grant_admin_address);
-        saveSubscriptionPayload.amount_per_period.push(Number(donation.grant.grant_donation_amount));
-        saveSubscriptionPayload.comment.push(comment);
-        saveSubscriptionPayload.confirmed.push(false);
-        saveSubscriptionPayload.contract_address.push(donation.grant.grant_contract_address);
-        saveSubscriptionPayload.contract_version.push(donation.grant.grant_contract_version);
-        saveSubscriptionPayload.denomination.push(tokenAddress);
-        saveSubscriptionPayload['gitcoin-grant-input-amount'].push(gitcoinGrantInputAmt);
-        saveSubscriptionPayload.grant_id.push(grantId);
-        saveSubscriptionPayload.split_tx_id.push(txHashes[i]);
-        saveSubscriptionPayload.sub_new_approve_tx_id.push(donation.tokenApprovalTxHash);
-        saveSubscriptionPayload.token_address.push(tokenAddress);
-        saveSubscriptionPayload.token_symbol.push(tokenName);
-      } // end for each donation
-
-      // to allow , within comments
-      saveSubscriptionPayload.comment = saveSubscriptionPayload.comment.join('_,_');
-
-      // Configure request parameters
-      const url = '/grants/bulk-fund';
-      const headers = {
-        'Content-Type': 'application/x-www-form-urlencoded; charset=UTF-8'
-      };
-      const saveSubscriptionParams = {
-        method: 'POST',
-        headers,
-        body: new URLSearchParams(saveSubscriptionPayload)
-      };
+        // if (json.failures.length > 0) {
+        //   // Something went wrong, so we create a backup of the users cart
+        //   await this.manageEthereumCartJSONStore(`${userAddress} - ${new Date().getTime()}`, 'save');
+        // }
+
+        // // Clear JSON Store
+        // await this.manageEthereumCartJSONStore(userAddress, 'delete');
+      } catch (err) {
+        // Something went wrong, so we use the manual ingestion process instead
+        console.error(err);
+        console.log('Standard contribution ingestion failed, falling back to manual ingestion');
+        await this.postToDatabaseManualIngestion(txHash, userAddress);
+      }
+    },
+
+    // Alternative to postToDatabase that uses the manual ingestion process
+    async postToDatabaseManualIngestion(txHash, userAddress) {
```

Comment:
anyway to DRY this with the one on the manual ingestion page?

Reply:
TL;DR yes, but wanted to get this out ASAP and not have to worry about potentially breaking manual ingestion page and re-testing

I did it this way because it keeps things simpler for now:
1. We ask the user to sign a different message, which is the way we inform them why we are signing a message
2. In this method, we'll only ever be ingesting Standard OR zkSync, but manual ingestion page supports both
3. Error handing is a bit different here, since it will tell users to use manual ingestion page if that fails